pages/_app.tsx

```diff
@@ -21,7 +22,9 @@ export default class App extends NextApp {
         <MDXProvider components={shortCodes}>
           <CacheProvider value={cache}>
             {globalStyles}
-            <Component {...pageProps} />
+            <AppLayout>
+              <Component {...pageProps} />
```

Comment:
The output is a nice, clean:

![Screen Shot 2020-08-04 at 12 36 05 AM](https://user-images.githubusercontent.com/36073/89260954-7cbc4c00-d5ea-11ea-99b5-868105881239.png)

Reply:
nice!